app/src/js/main.js

```diff
@@ -4,11 +4,11 @@ petDetectrApp.formSetup = function formSetup() {
     // Grab radio buttons and text nodes
     this.petStatus = document.querySelectorAll('[name="status"]');
     this.lastSeenText = document.querySelectorAll('.js-seen-found-text');
+    this.isChippedSelector = document.querySelectorAll('[name=is-chipped] + label');
```

Comment:
We should think of a way of switching these selectors from [attributes] to JS- hooks. Not important for this PR